src/hooks/useDebouncedCallback.ts

```diff
@@ -2,6 +2,7 @@ import { useCallback, useRef } from 'react';
 
 /**
  * Returns a callback that will only call the passed function when it has not been called for the specified time
+ *
```

Comment:
This is `jsdoc/tag-lines`. I'd like to enforce something like [PEP 257](https://peps.python.org/pep-0257/#multi-line-docstrings): a summary line followed by a blank line, followed by a longer description. eslint-plugin-jsdoc used to have a rule that did that but they recently deleted it https://github.com/gajus/eslint-plugin-jsdoc/issues/1031#issuecomment-1583129432